### PR TITLE
convert all the datetime attribute to Date object in JS

### PR DESCRIPTION
**Pull request by stormslowly**

### CODE REVIEW

**Comment by stormslowly** [comment]
the CI failed due to `npm`  failed ...

**Comment by tjwebb** [comment]
Ah, I think we can stop testing on node 0.8

**Comment by stormslowly** [comment]
how can i trigger the re-run and got merge ?

**Comment by dmarcelino** [comment]
@stormslowly, if you close the PR and re-open it will trigger a new build.

File: lib/database.js [hunk 1]
```diff
@@ -320,6 +320,21 @@ Database.prototype.select = function(collectionName, options, cb) {
...
+
+  var results = aggregate.results.map(function(item) {
+    dateTimeKeys.forEach(function(key) {
+      item[key] = new Date(item[key]);
+    });
+    return item;
+  });
+
   if(aggregate.error) return cb(aggregate.error);
   cb(null, aggregate.results);
```
Comment (marcelklehr) [inline lib/database.js]: Shouldn't this be `cb(null, results)`?

**Pull request closed without merging**